### DISCUSSION TITLE
Anyway for License Key?

### DISCUSSION THREAD

**Discussion by DeMoN100** [Ross-Patterson/Portals-Desktop-Organization#319] [General]
Hello this is a wonderful app is there anyway to participate in an event or giveaway for a license key? I share this app to many on open social networks everyone should know about portals!

**Comment by Ross-Patterson** [comment]
Hi @DeMoN100,
License keys are bought based on a pay what you want scheme, even 0.

Since there is a free option, any giveaway would be pretty pointless.

**Reply by DeMoN100** [reply]
I was just asking since I saw you gave a key to free to some brazillian user for no reason so I thought there were multiple opportunities. I didn't know it was selective to specific locations only. Thanks!

**Reply by Ross-Patterson** [reply]
It's not selective to specific locations only.

Just go to the website, enter 0 in the payment box and take a license.

That guy was having issues with the payment form, and so I did it on his behalf, that's all.

**Reply by antonvafsp** [reply]
> It's not selective to specific locations only.

Hi. I'm from Ukraine and I can't get a license under the 0 dollar scheme either too. I click the button and nothing happens. I tried to do it in different browsers, with and without VPN. 
I could help with the translation into Ukrainian, but I can't even switch the language in the app right now. 

Also, I found a bug with updating thumbnails and would like to test it in different modes before reporting a bug (if the photo file is large and takes a long time to load into the folder - the thumbnail never appears. You need to refresh the portal)

Although the free version suits me in general, I like to play with the settings.
